rebasebot/cli.py

```diff
@@ -181,6 +181,20 @@ def _parse_cli_arguments(testing_args=None):
         choices=["none", "soft", "strict"],
         help="Option that shows how to handle UPSTREAM tags in "
              "commit messages. (default: %(default)s)")
+    parser.add_argument(
+        "--allow-bot-squash",
+        action="store_true",
+        default=False,
+        required=False,
+        help="When enabled, the bot will be able to squash some bot's commits - provided a list of bot emails.",
+    )
+    parser.add_argument(
+        "--bot-emails",
```

Comment:
Just thought: would it make sense to provide defaults with ART's bot email list here?

Reply:
I think it would be better to be as it is. This flag is totally useless if used without the other ( `allow-bot-squash` ). It is run with with a script anyway afaik. So my take is that the user will always specify whose bot's commits should be taken care of.